extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_search.php

```diff
@@ -0,0 +1,33 @@
+<form class="wds-global-navigation__search" action="<?= Sanitizer::encodeAttribute( $model['module']['results']['url'] ); ?>">
+	<div id="searchInputWrapper" class="wds-dropdown wds-global-navigation__search-input-wrapper">
+		<label class="wds-global-navigation__search-label">
+			<?= DesignSystemHelper::getSvg(
+				'wds-icons-magnifying-glass',
+				'wds-icon wds-icon-small wds-global-navigation__search-label-icon'
+			) ?>
+			<input id="searchInput"
+				class="wds-global-navigation__search-input"
+				name="<?= Sanitizer::encodeAttribute( $model['module']['results']['param-name'] ); ?>"
+				<?php if ( !empty( $model['module']['suggestions'] ) ) : ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "suggestions" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)